src/panels/BruinPanel.ts

```diff
@@ -355,6 +355,25 @@ export class BruinPanel {
               "bruin"
             );
 
+            setTimeout(() => {
+              this._panel.webview.postMessage({
+                command: "runCompleted",
+                message: "",
+              });
+            }, 1500);
+            break;
```

Comment:
what is this setTimeout doing here?

Reply:
@y-bruin You can remove the `runCompleted` command as it doesn't send any information to the UI. The `setTimeout` was originally added to ensure enough time for the terminal to be created and the command to run, but it's no longer necessary.